ruby-challenges.rb

```diff
@@ -10,25 +18,84 @@
 reposts3 = 221
 # Expected output: '221 is odd'
 
+def even_odd num
+    if num % 2 == 0
+        "This is an even number"
+    elsif num % 2 != 0 
+        "This is an odd number"
+    else 
+        "This aint even a number dude"
+    end
+end
```

Comment:
Love it! But remember to look at your expected output vs your actual output. In this case they don't match.